Good-Bids/goodbids
# [#569] Updating the auction template pattern

# Summary

This updates the auction template pattern. Most of this is new content under the main bidding section. 

## Issues

* #569

## Testing Instructions

1. Create a new auction.
2. Make sure the new auction pattern shows up.


## Screenshots

<img width="359" alt="Screenshot 2024-03-04 at 3 02 33 PM" src="https://github.com/Good-Bids/goodbids/assets/91974372/7ac50e09-80a8-4095-9ea7-bb0a8eedacdc">

https://github.com/Good-Bids/goodbids/assets/91974372/8b5f0cd3-ac38-480f-971a-97990c322248
